### PR TITLE
Fixes #45, #46 - Fix a few consistency issues

### LINKED ISSUE #45: Update UA-CH token names
These should have a `Sec-CH` prefix and currently they do not.

https://wicg.github.io/client-hints-infrastructure/#client-hints-token-definition

I can send in a PR.

### PR DESCRIPTION
**Pull request by miketaylr**
Fixes a few spec inconsistencies, related to UA-CH tokens and prefixes in the Request processing algorithm.

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -244,10 +244,6 @@ if <var>request</var>'s <a for=request>header list</a>
    policy-controlled feature</a>, returns <code>false</code>, then skip the next steps and
    continue to the next <var>hintName</var>.
    [[!PERMISSIONS-POLICY]] [[!CLIENT-HINTS]]
-
-  <li><p>Set <var>hintName</var> to "Sec-" concatenated with <var>hintName</var>.
-  <div class=issue>We need to figure out if we really want a `Sec-` prefix, and if so also exempt it from CORS.</div>
```
Comment (miketaylr) [inline index.bs:249]: Should we file an issue related to the CORS question, assuming it's still an open issue?

**Comment by miketaylr** [comment]
This removes the `Set <var>hintName</var> to "Sec-" concatenated with <var>hintName</var>.` step from the request processing algorithm, which makes it reflect the current implementation in Chromium. However, I'm not sure if there are intentions for all CH tokens to have a `Sec-CH` prefix (eventually).

If so, I can do something slightly different and add some language to that effect and special case the set of "legacy" tokens (Save-Data, DRP, Viewport-Width, Width, Device-Memory, Downlink, RTT, ECT).

PTAL @yoavweiss, thanks!

File: index.bs (hunk 1 above)
Comment (yoavweiss) [reply index.bs:249]: There's an open [PR](https://github.com/whatwg/fetch/pull/1000) on Fetch.

**Review by yoavweiss — APPROVED** [review]
LGTM

**Comment by miketaylr** [comment]
Thanks, @yoavweiss. I filed https://github.com/WICG/client-hints-infrastructure/issues/49 so feel free to merge here when you have a chance.

**Pull request merged**